process.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "path/filepath"
+    "io/ioutil"
+    "strings"
+    "strconv"
+    "sync"
+)
+
+func main() {
+    args := os.Args
+    sourceDir, err := filepath.Abs(args[1])
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
+        os.Exit(0);
+    }
+    destDir, err := filepath.Abs(args[2])
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
+        os.Exit(0);
+    }
+
+    if _, err := os.Stat(destDir); os.IsNotExist(err) {
+        fmt.Println(destDir)
+        os.Mkdir(destDir, 0777)
+    }
+
+    files, err := ioutil.ReadDir(sourceDir)
+    if err != nil {
+        fmt.Printf("error: %v\n", err)
+        os.Exit(0);
+    }
+
+    var wg sync.WaitGroup
+
+    countFileLines := func(fileName, sourceDir, destDir string, wg *sync.WaitGroup) {
+        data, err := ioutil.ReadFile(filepath.Join(sourceDir,fileName))
+        if err != nil {
+            fmt.Printf("error: %v\n", err)
+            os.Exit(0);
+        }
+        lineCount := 1
```

Comment:
`chank` may be a misleading name, `byte` is better here. Because a chunk may be a one byte maybe more, and we know for sure here that iteration goes byte by byte.

You can also use `len(bytes.Split(data, []byte{'\n'}))` instead. It's less efficient but simpler to understand